app/views/cart/show.html.erb

```diff
@@ -3,6 +3,7 @@
       <p>Your cart is empty</p>
     <% else %>
     <p> <%=link_to "Empty Cart", "/cart", method: :delete %> </p>
+    <p> <%=link_to "Checkout", "/order", method: :post %> </p>
```

Comment:
Now we have a checkout link in our little shop! It makes sense the link has a new route to an order index page.

We could also move checkout link to bottom of the page later (when styling the page)